addon/-legacy-private/system/relationships/state/relationship.js

```diff
@@ -87,10 +94,9 @@ export default class Relationship {
       true when
         => !internalModel.isNew() on initial setup
         => an inverse has been unloaded
-        => relationship.reload() has been called
         => we get a new link for the relationship
      */
-    this.relationshipIsStale = !internalModel.isNew();
+    this.relationshipIsStale = !this.isNew;
```

Comment:
Can you add a comment as to what the difference between forcing a reload and a stale relationship is

Reply:
👍